src/pages/UserTable/UserTable.tsx

```diff
@@ -361,9 +361,9 @@ const UsersTable = () => {
       <Title level={4} style={{ textAlign: "left", margin: 10 }} underline={true}>Загальна кількість користувачів: {total}</Title>
       <div className={classes.searchContainer}>
         <div className={classes.filterContainer}>
-          <Form form={form} onFinish={handleFilter} style={{ height: "20px" }}>
+          <Form form={form} onFinish={handleFilter}>
             <Row style={{ flexFlow: "nowrap" }}>
-              <Col span={20}>
+              <Col style={{ width: "88%"}}>
```

Comment:
It's bad practice to write css code inside the component. It would be better to create a new class inside another css file